package.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
I'm seeing a lot of properties that probably don't need to be here.  https://github.com/blackbaud/skyux-indicators/blob/master/package.json as a comparison..